frameworks/hdfs/src/main/dist/svc.yml

```diff
@@ -51,6 +51,22 @@ pods:
           {{/TLS_ENABLED}}
         env:
           JOURNALNODE: true
+        # The existence of the journal-data/hdfs/current/committed-txid file indicates HDFS has already been deployed once
```

Comment:
I would extend this comment a bit more future readers.  It only describes the simple `if` case, might want to describe what the `else` clause is doing too.